var/spack/repos/builtin/packages/py-nilearn/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyNilearn(PythonPackage):
+    """Statistical learning for neuroimaging in Python."""
+
+    homepage = "http://nilearn.github.io/"
+    pypi     = "nilearn/nilearn-0.7.1.tar.gz"
+
+    version('0.7.1', sha256='8b1409a5e1f0f6d1a1f02555c2f11115a2364f45f1e57bcb5fb3c9ea11f346fa')
+
+    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+
+    variant('plotting', default=True, description='Enable plotting functionalities')
```

Comment:
I would disable this by default, it's usually good to minimize the number of dependencies.